.github/workflows/ci.yml

```diff
@@ -23,9 +23,16 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        scala: [2.13.7]
+        scala: [2.13.8]
         java: [temurin@8, temurin@11]
     runs-on: ${{ matrix.os }}
+    services:
+      mysql:
+        image: mysql
```

Comment:
That's a good question—I wonder if we can add a matrix entry for different MySQL versions and have it run the test suite against all of them.

Just to keep things moving, I'm also ok with punting on that for now, moving forward with `image: mysql`, and following up later.

Reply:
Adding a matrix worked, but since roles weren't added to MySQL until version 8, the current implementation won't work on anything older than that. I think I'll leave the matrix in place but only with `mysql:8`, unless we really need to support older versions too.